buildtools/install_sqlite.bat

```diff
@@ -0,0 +1,60 @@
+@rem Control variables
+@rem - SQLite {Note: `powershell` cannot `expand-archive` to `C:\Program Files (x86)`}
+@set sqlite_zip=sqlite-dll-win64-x64-3310100.zip
+@set sqlite_folder=%USERPROFILE%\.sqlite
+@set sqlite_runtime=sqlite3.dll
+
+@rem Determine if running as administrator
+@call :TEST_ADMINISTRATOR
+@if [%errorlevel%]==[10101] goto :END
+
+@rem Install dependencies
+@call :INSTALL_SQLITE
+@goto END:
+`
+:INSTALL_SQLITE
+@rem Download install file
+@powershell wget https://www.sqlite.org/2020/%sqlite_zip% -outfile "%TEMP%\%sqlite_zip%"
```

Comment:
Might be nice to echo something like "Downloading and installing sqlite..." just so that the user knows that something is happening.

Reply:
Good idea, will do